upgrade_scripts/pg_sphere--1.3.1--1.3.2.sql.in

```diff
@@ -16,3 +16,12 @@ $$;
 -- remove legacy spellings of operators
 DROP OPERATOR IF EXISTS @(bigint, smoc);
 DROP OPERATOR IF EXISTS @(spoint, smoc);
+
+CREATE FUNCTION spoly(spoint[])
+  RETURNS spoly
+  AS 'MODULE_PATHNAME', 'spherepoly_from_point_array'
+  LANGUAGE 'c'
+  IMMUTABLE STRICT PARALLEL SAFE;
+
+COMMENT ON FUNCTION spoly(spoint[]) IS
+  'Create spoly from an array of points.';
```

Comment:
Change comment to `creates spoly from an array of points` (no capitalization, no period, and "creates" not "create") in order to match the other comments on the other functions.